network/topics/scoring.go

```diff
@@ -66,6 +70,8 @@ func scoreInspector(logger *zap.Logger, scoreIdx peers.ScoreIndex) pubsub.Extend
 			logger.Debug("peer scores", fields...)
 
 			metricPubsubPeerScoreInspect.WithLabelValues(pid.String()).Set(peerScores.Score)
+			metricPubSubPeerP4Score.WithLabelValues(pid.String()).Set(p4ScoreSquaresSum)
```

Comment:
This one is not being reset like the one above, so it means it can grow infinitely no?